test/int/fixtures/defaultFixture.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { IFixture } from './fixture';
+import { ExtendedDebugClient } from 'vscode-chrome-debug-core-testsupport';
+import * as testSetup from '../testSetup';
+import { IBeforeAndAfterContext } from 'mocha';
+
+/**
+ * Default set up for all our tests. We expect all our tests to need to do this setup
+ * which includes configure the debug adapter, logging, etc...
+ */
+export class DefaultFixture implements IFixture {
+    private constructor(public readonly debugClient: ExtendedDebugClient) {
+        // Running tests on CI can time out at the default 5s, so we up this to 10s
+        debugClient.defaultTimeout = 15000;
+    }
+
+    /** Create a new fixture using the provided setup context */
+    public static async create(context: IBeforeAndAfterContext): Promise<DefaultFixture> {
+        return new DefaultFixture(await testSetup.setup(context));
+    }
+
+    /** Create a new fixture using the full title of the test case currently running */
+    public static async createWithTitle(testTitle: string): Promise<DefaultFixture> {
+        return new DefaultFixture(await testSetup.setupWithTitle(testTitle));
```

Comment:
I noticed `setupWithTitle` doesn't yet exist on `testSetup`, I'm guessing that's coming in the next PR?

Reply:
That's totally not on purpose. I don't know how that compiled...